flex/engines/http_server/types.h

```diff
@@ -59,6 +59,8 @@ using admin_query_result = payload<gs::Result<seastar::sstring>>;
 // url_path, query_param
 using graph_management_param =
     payload<std::pair<seastar::sstring, seastar::sstring>>;
+using graph_management_query_param =
```

Comment:
Why do we need a map?

Reply:
Because the get request is for a map, not a json.